shopinvader_search_engine/README.rst

```diff
@@ -2,91 +2,58 @@
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
 
-==============
-{module_title}
-==============
+=========================
+Shopinvader Search Engine
+=========================
+
+This module is technical module for search engines.
```

Comment:
@hparfr  Disclaimer: I'm not the best one to write clear description for modules. Nevertheless what do you think about 
```
This module is a technical module to implement batch export of data to external indexation services.
The first concrete implementation allows to export data to Algolia.
```